docs/Migrations.md

```diff
@@ -46,4 +46,112 @@ Migrations which are Node modules should export a function which will be called
 
 You can execute SQL using `tx.executeSql(query)`, however note that the model for which the migration is running will *not* be available under `sbvrUtils.api`, as it has not executed yet.
 
-You should return a promise so pinejs can wait for completion/errors.
\ No newline at end of file
+You should return a promise so pinejs can wait for completion/errors.
+
+
+## Async Migrations
+
+Migrations that may lock tables for a long time, eg. column renames on large tables, can be run as async migrations.
```

Comment:
I think column renames only affect the catalog. Maybe we should say something like:
```suggestion
Migrations that may lock tables for a long time, eg. migrating data between columns on large tables, can be run as async migrations.
```